compiler/one-cmds/onnx_legalizer.py

```diff
@@ -97,7 +154,7 @@ def make_split(self, input, split_sizes, axis):
 
         Args:
             input (str): name of input tensor
-            split_sizes (list): list of split sizes
+            split_sizes (list of int): list of split sizes
```

Comment:
this block in in single quote `'''` but others in double. I'm not used to python docs, is this OK or should we change this too?

Reply:
will replace, this is leftovers from previous iteration =)